src/js/BookReader/Mode2Up.js

```diff
@@ -535,6 +535,8 @@ export class Mode2Up {
    * @param {number} newIndexR
    */
   flipLeftToRight(newIndexL, newIndexR) {
+    this.br.refs.$brContainer.addClass("BRpageFlipping");
+    if (this.br.enableTextSelection) this.br.textSelectionPlugin.hideTextLayer(this.br.refs.$brContainer);
```

Comment:
Add this to the css file instead of calling a function; something like `.BRpageFlipping svg { display: none; }`

Reply:
done